ark/utils/plot_utils.py

```diff
@@ -6,11 +6,42 @@
 import skimage.io as io
 import matplotlib as mpl
 import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
 
 from skimage.segmentation import find_boundaries
 from skimage.exposure import rescale_intensity
 
 
+# plotting functions
+
+def plot_clustering_result(img_list, titles, cmap='tab20', figsize=(10, 10)):
```

Comment:
Add an argument called `save_dir` if the user also wants to save each image into a directory (default argument should be `None`). You can use `misc_utils.save_figure` to do this:

```
if save_dir is not None:
    misc_utils.save_figure(save_dir, "file_name.png")
```